app/aws/s3.py

```diff
@@ -107,3 +107,20 @@ def upload_letters_pdf(reference, crown, filedata):
 
     current_app.logger.info("Uploading letters PDF {} to {}".format(
         upload_file_name, current_app.config['LETTERS_PDF_BUCKET_NAME']))
+
+
+def get_list_of_files_by_suffix(bucket_name, subfolder='', suffix='', last_modified=None):
```

Comment:
last_modified parameter name is not clear (before-after-equal?) - I would have called it modified_after